Document user recent avatars (#341)
discord-userdoccers/discord-userdoccers: pages/resources/user.mdx

| avatar?                   | ?[image data](/reference#cdn-data) | The user's avatar; can be animated when the user has an applicable premium (Nitro) plan                                                                                   |
| avatar_decoration_id?     | ?snowflake                         | The ID of the user's avatar decoration                                                                                                                                    |
| avatar_decoration_sku_id? | ?snowflake                         | The SKU ID of the user's avatar decoration                                                                                                                                |

> | avatar_description?       | ?string                            | The user's avatar description usually in format "{filename}, added {date}" (max 1024 characters)                                                                          |

Reviewer: ```suggestion
| avatar_description?       | ?string                            | The description of the new user avatar, usually in the format "{filename}, added {date}" (max 1024 characters)                                                                          |
```
this is also missing the `avatar_id` field, also I would move this under `avatar?` so it's `avatar avatar_id avatar_description`

| ------------ | ------- | ----------------------------------------- |
| global_name? | ?string | The user's display name (1-32 characters) |

<RouteHeader method="GET" url="/users/@me/avatars">
  Get Recent Avatars
</RouteHeader>

> Returns the user's recent avatars. Only 2 avatars will be returned for users without a Nitro subscription, otherwise 6 recent avatars will be returned.

Reviewer: ```suggestion
Returns the user's recent avatars. For premium users, the six most recent avatars will be returned. Otherwise, only two will be returned.
```

###### Response Body

| Field   | Type                                        | Description                     |
| ------- | ------------------------------------------- | ------------------------------- |

> | avatars | array[[avatars](#avatars-structure) object] | The recent avatars for the user |

Reviewer: ```suggestion
| avatars | array[[avatar](#avatar-structure) object] | The recent avatars for the user |
```

> ###### Avatars Structure

Reviewer: ```suggestion
###### Avatar Structure
```
why plural

| Field        | Type    | Description                                |
| ------------ | ------- | ------------------------------------------ |

> | id           | string  | The avatar IDs                             |

Reviewer: ```suggestion
| id           | string  | The avatar ID                             |
```
same here

> | storage_hash | string  | The hash of the avatar                     |

Reviewer: ```suggestion
| storage_hash | string  | The [avatar hash](/reference#cdn-formatting)                    |
```

> | description  | ?string | The description when the file was uploaded |

Reviewer: ```suggestion
| description  | ?string | The description specified when the avatar was uploaded |
```
